### DISCUSSION TITLE
Welcome to group3-project Discussions!

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by matty10s** [comment]
GitHub MMC5277 Repo has been created. :)

**Comment by matty10s** [comment]
Feel free to post discussion comments here for our project if you would like to?  Optional.  or can just email. Which ever is easier for you 👍

**Comment by tay-baum** [comment]
I think this makes sense for keeping everything in one place! Then we also don't have to worry about accidentally not "replying all" 😊
